src/test/java/org/opensearch/security/dlic/rest/api/AllowlistApiTest.java

```diff
@@ -97,7 +97,9 @@ public void testResponseDoesNotContainMetaHeader() throws Exception {
 
         rh.sendAdminCertificate = true;
         RestHelper.HttpResponse response = rh.executeGetRequest(ENDPOINT);
-        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+
+        // Replaced Assert.asserEquals to assertThat
```

Comment:
Lets remove these comments; `git diff` `git blame` and `git log -- src/test/java/org/opensearch/security/dlic/rest/api/AllowlistApiTest.java` are enough to track the were and why of a given change.